compiler/src/iree/compiler/Dialect/Util/Analysis/Constant/ConstExpr.cpp

```diff
@@ -502,6 +504,15 @@ void ConstExprHoistingPolicy::makeDecision(
     return;
   }
 
+  // If the op explicitly says no, we disable the hoisting.
+  if (auto hoistableOp =
+          dyn_cast<Util::HoistableOpInterface>(info->getOperation())) {
+    if (!hoistableOp.isHoistableOp()) {
+      decision->disableHoist();
+      return;
+    }
+  }
+
```

Comment:
I should probably split out the change to the other PR. It is just testing for now.